drivers/SmartThings/matter-window-covering/fingerprints.yml

```diff
@@ -5,7 +5,12 @@ matterManufacturer:
     vendorId: 0x130A
     productId: 0x55
     deviceProfileName: window-covering-profile
-
+#Zemismart
+- id: 5020/65534
```

Comment:
Indent this line to follow the formatting in the other fingerprints. Also this should be a string (should be surrounded with quotes "").

Where did this ID come from? I am wondering if we can change it to something more human readable? `5020/65534` doesn't really tell us much about what this device is. See above where the id is "Eve MotionBlinds", which makes it clear what we are looking at.

Reply:
The partner provided that information, I will need to follow up with the certification tool team regarding that. Thanks Cooper. I made another commit.